apps/vaporgui/VStringLineEdit.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <iostream>
+
+#include <string>
+#include <sstream>
+#include <iomanip>
+
+#include "VContainer.h"
+
+class QStringLineEdit;
+
+//! \class VStringLineEdit
+//! \ingroup Public_GUI
+//! \brief A wrapper for a QLineEdit that handles user input of type string, 
+//! and provides Vapor's standard setters, getters, and signals
+
+class VStringLineEdit : public VContainer {
+    Q_OBJECT
+
+    public:
+        VStringLineEdit( std::string value = "" );
+
+        //! Set the current string value in the line edit
+        virtual void SetValueString( std::string value );
```

Comment:
The SetString functions can be disabled for the numeric widgets with delete.

Reply:
This function is actually used in the numeric line edits to apply a formatted number to the VLineEdit.

Reply:
Fixed.